### PR TITLE
Add chocolatey package install instructions

### PR DESCRIPTION
**Pull request by doggy8088**
https://github.com/go-gitea/gitea/issues/10800

### CODE REVIEW

File: docs/content/doc/installation/from-package.en-us.md [hunk 1]
```diff
@@ -38,9 +38,11 @@ config is found in **/etc/gitea/app.ini**
...
 ## Windows
 
-There are no published packages for Windows. This page will change when packages are published,
-in the form of `MSI` installers or via [Chocolatey](https://chocolatey.org/). In the meantime, follow
-the [deployment from binary]({{< relref "from-binary.en-us.md" >}}) guide.
+There is a [Gitea](https://chocolatey.org/packages/gitea) package for Windows by [Chocolatey](https://chocolatey.org/).
+
+```sh
+choco install gitea
+```
```
Comment (zeripath) [inline docs/content/doc/installation/from-package.en-us.md:46]:
```suggestion

Or follow the [deployment from binary]({{< relref "from-binary.en-us.md" >}}) guide.
```

**Review by zeripath — APPROVED** [review]
Minor nit but otherwise ok.

I think we need an @go-gitea/owners review before merging this.

**Comment by techknowlogick** [comment]
I'm fine with this change, as we have other 3rd party repos that we don't manage on this page as well.

Unrelated to the change though I think the gitea project should take over responsibility for the choco package.

**Comment by techknowlogick** [comment]
@doggy8088 👋 Hey, I see you are the current publisher of the chocolatey package, would you be open to transferring it to the Gitea project? I'm currently one of the elected owners of Gitea and would be able to assist with the transfer of the package. My user on chocolatey is also techknowlogick

**Comment by doggy8088** [comment]
I'm ok with that.

**Comment by doggy8088** [comment]
@techknowlogick I already setup CI/CD for the gitea Chocolatey package. Should I stop auto publishing to Chocolatey someday?

**Comment by techknowlogick** [comment]
Thanks @doggy8088 :)

We have access to windows servers for our CI and will be able to take over publishing. We appreciate your past work in publishing the package.

**Pull request merged**